go-controller/pkg/crd/clusternetworkconnect/v1/types.go

```diff
@@ -0,0 +1,189 @@
+/*
+Copyright 2025.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/crd/types"
+)
+
+// ClusterNetworkConnect enables connecting multiple User Defined Networks
+// or Cluster User Defined Networks together.
```

Comment:
for now sounds like only UDNs can be interconnected OR CUDNs, but not the mix?
Maybe replacing "or" with "and" will be better?